dissect/target/plugins/os/unix/linux/processes.py

```diff
@@ -0,0 +1,55 @@
+from typing import Iterator
+
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+ProcProcessRecord = TargetRecordDescriptor(
+    "unix/linux/proc/processes",
+    [
+        ("datetime", "ts"),
+        ("string", "name"),
+        ("string", "state"),
+        ("varint", "pid"),
+        ("datetime", "runtime"),
+        ("varint", "ppid"),
+        ("string", "parent"),
+    ],
+)
+
+
+class ProcProcesses(Plugin):
+    def check_compatible(self) -> None:
+        if not self.target.proc:
+            raise UnsupportedPluginError("No /proc directory found")
+
+    @export(record=ProcProcessRecord)
+    def processes(self) -> Iterator[TargetRecordDescriptor]:
+        """This plugin yields the processes available in /proc and the stats associated with them.
+
+        There is a numerical subdirectory for each running process; the subdirectory is named by the process ID.
+        Each /proc/[pid] subdirectory contains various pseudo-files.
```

Comment:
```suggestion
        """This plugin yields the processes available in ``/proc`` and the stats associated with them.

        There is a numerical subdirectory for each running process; the subdirectory is named by the process ID.
        Each ``/proc/[pid]`` subdirectory contains various pseudo-files.
```
Also note that you use `[pid]` here again.